buf/registry/module/v1/label_service.proto

```diff
@@ -115,6 +115,8 @@ message ListLabelsRequest {
   //
   // TODO: Add custom CEL validation to validate the status field is one of DISABLED, PASSED, APPROVED.
   repeated CommitCheckStatus commit_check_statuses = 5 [(buf.validate.field).repeated.items.enum.defined_only = true];
+  // Only return Labels with a name that contains this string.
+  string name = 6 [(buf.validate.field).string.max_len = 250];
```

Comment:
Could also make this more generically named `query`.

Reply:
Would it make sense to have a `ListLabelsFilter` message with all the possible filters for a given request?

```proto
ListLabelsFilter filters = 6;
```

Reply:
In a live convo, decided that we wanted to keep the datastructure flat and instead to call it `name_query`.